tests/ballerina-compiler-api-test/src/test/resources/test-src/expressions_test.bal

```diff
@@ -358,3 +358,10 @@ function fooFn() returns future<int> {
 function barFn() returns int {
     return 10;
 }
+
+function testRegexp() {
+    var reA = re `[a-z0-9]+`;
+    var reB = re ``;
+    var reC = re `a`;
+    var reD = re `[a|b]+c`;
```

Comment:
```suggestion
    _ = re `[a-z0-9]+`;
    _ = re ``;
    _ = re `a`;
    _ = re `[a|b]+c`;
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/39466/commits/1d6a475506892f8791a542f5b35d14d606218d7f.